src/tokens/PositionItem.tsx

```diff
@@ -77,94 +73,19 @@ export const PositionItem = ({ position }: { position: Position }) => {
   )
 }
 
-export const TokenBalanceItem = ({
```

Comment:
No longer needed used after #5016 , knip didn't catch it because it was referenced in tests